framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -283,6 +284,12 @@ public void updateFromFieldAssignment(
                             + lhs.getClass());
         }
 
+        // Do not attempt to infer types for temporary variables.
```

Comment:
Can you define the term "temporary variable"?  Do you mean "compiler-generated"?

Reply:
Yes, that's what I was referring to. I will change this documentation to refer to variables with invalid names, and give compiler-generated temporary variables as an example.